spec/models/dataset_spec.rb

```diff
@@ -12,17 +12,6 @@
     expect(dataset.errors[:summary]).to include "Please provide a summary"
   end
 
-  it "validates title format" do
-    dataset = Dataset.new(title: "[][]")
```

Comment:
OK. I didn't connect titles and slugs when I made the change. @nicholsj is a valid title just a string of words (i.e. no special characters except whitespace)?

Reply:
@rossjones @benthorner Ok, I'm going to have to leave it to devs to agree a way forward on this. As it was explained to me, it was simply checking that the title had 3 characters - and that seemed pointless to me. 

Is this another case where we should generate a new card to explore/fix an issue, rather than wrapping it up in another PR which means it gets blocked?